scripts/release.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+set -xe
+
+SRC_DIR="${SRC_DIR:-$PWD}"
+BUILDFLAGS="-a --ldflags '-extldflags \"-static\"'"
+
+TAG=$(git describe --exact-match --abbrev=0) || TAG=$(git describe --tags)
```

Comment:
I think this could just be changed to `TAG=$(git describe --tags)` as that should work in all cases. I'm not sure why we have the `--exact-match --abbrev=0` bit